helperApp/plaid/plaid.js

```diff
@@ -1,25 +1,16 @@
+require('./plaid.scss');
```

Comment:
I first thought that this would include the CSS inside the Javascript, but it nicely generates `plaid.css`. Such magic!